forms-flow-theme/scss/_modal.scss

```diff
@@ -856,3 +856,71 @@
    }
  }
 }
+
```

Comment:
line 859 to 926 can be moved to fileUpload.scss as we have a separate file for import operations